com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that cased data loss when the AlembicStreamPlayer was reset.
```

Comment:
Typo. Should be "caused"

Reply:
suggestion:
`- Fixed a bug that was causing data loss when resetting the AlembicStreamPlayer.`